inference-engine/src/vpu/graph_transformer/src/middleend/passes/adjust_data_batch.cpp

```diff
@@ -201,10 +206,33 @@ void PassImpl::wrapInLoop(const Model& model, const StageList& subgraph) {
         }
     }
 
+    const auto getBatchDataIndex = [](DataVector& stageInputs, const Data& data) {
+        auto i = 0;
+        for (; i < stageInputs.size(); i++) {
+            if (stageInputs[i] == data)
+                break;
+        }
+        if (i == stageInputs.size())
+            stageInputs.push_back(data);
+        return i;
+    };
```

Comment:
A bit strange lambda to me: it is getter what makes me thinks it does not modify anything, but it modifies some vector - adds new input for a stage??

Also, is there a way to reuse `std::find` algo?